### PR TITLE
Update to Java 11

### PR DESCRIPTION
**Pull request by rtyley**
This satisfies one of the updated requirements of Panda [v7](https://github.com/guardian/pan-domain-authentication/releases/tag/v7.0.0) - as of https://github.com/guardian/pan-domain-authentication/pull/150 it requires Java 11.

See also:

* https://github.com/guardian/maintaining-scala-projects/issues/3

### Testing
Successfully [deployed](https://riffraff.gutools.co.uk/deployment/view/4d33116d-b01e-45f0-a56e-2c70cbae050a) to TEST at 264b2bd:

https://github.com/user-attachments/assets/ed0db6c1-e777-4744-9b55-64f71582f640

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -14,6 +14,18 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.2")
 
-// needed by Snyk to accurately report vulnerabilities
-// https://docs.snyk.io/scan-application-code/snyk-open-source/snyk-open-source-supported-languages-and-package-managers/snyk-for-scala
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```
Comment (rtyley) [inline project/plugins.sbt:19]: See:

* https://github.com/guardian/maintaining-scala-projects/issues/12

**Comment by Fweddi** [comment]
Is this PR up for review? I notice some failed deploys to TEST recently: https://riffraff.gutools.co.uk/deployment/history?projectName=media-service%3A%3Agrid%3A%3Aall&page=1.

**Comment by rtyley** [comment]
> Is this PR up for review? I notice some failed deploys to TEST recently: https://riffraff.gutools.co.uk/deployment/history?projectName=media-service%3A%3Agrid%3A%3Aall&page=1.

Yeah, apologies, I'm pretty sure I've fixed all those issues, but I haven't managed to find a good time to do a deploy to Grid TEST since I did all the fixes (and clearing up a bad deploy on Grid took me a lot of time, as there were a lot of ASGs to fix one by one - is there a better way to do this?!) - just waiting for Tom to finish on Grid TEST.

File: .tool-versions [hunk 2]
```diff
@@ -0,0 +1 @@
+java corretto-11.0.24.8.1
```
[the remark this answers is not in the document]
Comment (rtyley) [reply .tool-versions:1]: This is an [`asdf`](https://asdf-vm.com/)-formatted `.tool-versions` file - I'm trying to nudge peo0ple towards standardising on it (see https://github.com/guardian/gha-scala-library-release-workflow/pull/36) and thought I'd add it in here. It does a very similar job to the [.java-version](https://github.com/guardian/grid/blob/main/.java-version) file that's already in the project, which I think is a [jEnv](https://www.jenv.be/) thing.

**Comment by rtyley** [comment]
I've managed to [deploy](https://riffraff.gutools.co.uk/deployment/view/38bfbdce-84f7-4d92-9cb2-9db7c29be0c3) _most_ of this to TEST, but the two services (`image-loader` & `cropper`) fail to deploy - although they use Java (and so now need Java 11) they _don't_ use the `editorial-tools-focal-javaX-ARM-WITH-cdk-base` Amigo image - they use [`grid-imaging-ARM`](https://amigo.gutools.co.uk/recipes/grid-imaging-ARM):

https://github.com/guardian/grid/blob/d2ec798aa89fc7a0b73efcfae9c9271d9b172e14/riff-raff.yaml#L89-L96

I guess I should clone `grid-imaging-ARM` and make `grid-imaging-java11-ARM` - that sound ok @twrichards ?

**Comment by rtyley** [comment]
Finally successfully [deployed](https://riffraff.gutools.co.uk/deployment/view/4d33116d-b01e-45f0-a56e-2c70cbae050a) this to TEST, and it looks good:

https://github.com/user-attachments/assets/ed0db6c1-e777-4744-9b55-64f71582f640

...so I think this is now ready to review!

**Review by Fweddi — APPROVED** [review]

**Comment by rtyley** [comment]
This has [deployed](https://riffraff.gutools.co.uk/deployment/view/e8ec0420-0344-468c-866d-69252212b658) successfully - and the Grid looks good:

image

**Pull request merged**